src/components/interaction-designer/toolbar/ErrorNotifications.vue

```diff
@@ -111,26 +111,19 @@ export class ErrorNotifications extends mixins(Routes, Lang) {
     return pickBy(this.validationStatuses, (value: IValidationStatus, key) => key.includes('block/') && blocksMap[key])
   }
 
-  get resourceValidationStatusesForActiveFlow(): Dictionary<IValidationStatus> {
-    return pickBy(this.validationStatuses, (status, key) => {
-      const resourceUuid = replace(key, 'resource/', '')
-      return this.resourceUuidsOnActiveFlow.includes(resourceUuid) && !get(this.validationStatuses, `resource/${resourceUuid}`)?.isValid
-    })
-  }
-
```

Comment:
I think the issue here was using `this` in an arrow function in a class's method which gave it a wrong scope (`this` was undefined)